test-engineering/contract-tests/client/tests/test_merino.py

```diff
@@ -6,6 +6,7 @@
 from typing import Dict, List, Set, Tuple
 
 import pytest
+import time
```

Comment:
Suggest to group the std library imports.

```suggestion
import time
from typing import Dict, List, Set, Tuple

import pytest
```

Reply:
Good point, done!